src/ploomber/tasks/notebook.py

```diff
@@ -669,7 +691,10 @@ def __init__(self,
                  nbconvert_export_kwargs=None,
                  local_execution=False,
                  check_if_kernel_installed=True,
-                 debug_mode=None):
+                 debug_mode=None,
+                 extract_up=False,
+                 extract_prod=False
```

Comment:
we dont need this in the class constructor right? these will always be False when initializing the class